protocol/rpcprovider/rpcprovider_server.go

```diff
@@ -180,7 +183,12 @@ func (rpcps *RPCProviderServer) craftChainMessage() (chainMessage chainlib.Chain
 
 // function used to handle relay requests from a consumer, it is called by a provider_listener by calling RegisterReceiver
 func (rpcps *RPCProviderServer) Relay(ctx context.Context, request *pairingtypes.RelayRequest) (*pairingtypes.RelayReply, error) {
-	grpc.SetTrailer(ctx, metadata.Pairs(chainlib.RpcProviderUniqueIdHeader, rpcps.providerUniqueId))
+	// count the number of simultaneous relay calls
+	rpcps.providerLoadManager.addRelayCall()
+	defer rpcps.providerLoadManager.removeRelayCall()
+	provider_relay_load := strconv.FormatFloat(rpcps.providerLoadManager.getProviderLoad(), 'f', -1, 64)
```

Comment:
no snake cases - please fix all of them :)

Reply:
Fixed in [6db7dd3](https://github.com/lavanet/lava/pull/1720/commits/6db7dd33d05f92ed3d3feeb6c7106b7d33d1ccc6)